app/assets/stylesheets/application.css.scss

```diff
@@ -30,11 +31,11 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 
 
 .post {
-  background-color: $post-bg-color;
-  border-radius: 10px;
-  list-style: none;
-  margin: 20px 10px;
+  background-color: $white;
+  border-radius: 5px;
+  margin: 10px;
   padding: 5px 20px;
+  border: 1px solid #e1e8ed;
```

Comment:
Color literals like `#e1e8ed` should only be used in variable declarations; they should be referred to via variable everywhere else.